apps/widgattbat/widget.js

```diff
@@ -0,0 +1,16 @@
+(() => {
+  function advertiseBattery() {
+    Bangle.bleAdvert[0x180F] = [E.getBattery()];
+    NRF.setAdvertising(Bangle.bleAdvert);
+  }
+
+  if (!Bangle.bleAdvert) Bangle.bleAdvert = {};
+  setInterval(advertiseBattery, 60 * 1000);
+  advertiseBattery();
+
+  WIDGETS["gattbat"]={
+    area:"tl",
+    width:0,
+    draw:function() {}
+  };
```

Comment:
Does it need to declare this stuff if it isn't drawing anything to the screen?

Reply:
Nope, not at all... Actually that's probably the easiest option.